<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainLoja</class>
 <widget class="QMainWindow" name="MainLoja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estoque em Loja</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="itensLayout">
      <item>
       <layout class="QHBoxLayout" name="produtosLayout">
        <item>
         <layout class="QVBoxLayout" name="livrosLayout">
          <item>
           <widget class="QLabel" name="livrosLabel">
            <property name="text">
             <string>LIVROS</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="livrosWidget">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="tabKeyNavigation">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>NOME</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>PREÇO</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>AUTOR</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="cdsLayout">
          <item>
           <widget class="QLabel" name="cdsLabel">
            <property name="text">
             <string>CDS</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="cdsWidget">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="tabKeyNavigation">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>NOME</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>PREÇO</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>N° FAIXAS</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dvdsLayout">
          <item>
           <widget class="QLabel" name="dvdsLabel">
            <property name="text">
             <string>DVDS</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="dvdsWidget">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="tabKeyNavigation">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>NOME</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>PREÇO</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>DURAÇÃO</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionLer"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuItem">
    <property name="title">
     <string>Item</string>
    </property>
    <addaction name="actionIncluirLivro"/>
    <addaction name="actionIncluirCD"/>
    <addaction name="actionIncluirDVD"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuItem"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLer">
   <property name="text">
    <string>Ler...</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar...</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionIncluirLivro">
   <property name="text">
    <string>Incluir Livro...</string>
   </property>
  </action>
  <action name="actionIncluirCD">
   <property name="text">
    <string>Incluir CD...</string>
   </property>
  </action>
  <action name="actionIncluirDVD">
   <property name="text">
    <string>Incluir DVD...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
